protocols/relay/src/behaviour/handler.rs

```diff
@@ -18,10 +18,11 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-use crate::behaviour::CircuitId;
-use crate::copy_future::CopyFuture;
-use crate::proto;
-use crate::protocol::{inbound_hop, outbound_stop};
```

Comment:
If possible, it would be great to avoid diffs like these. The `crate::` and `std::` imports seem to just have been flipped but are largely unchanged.